### PR TITLE
Create migration to remove ltss7 from products

### PR DESCRIPTION
**Pull request by Adnilson**
## Description

Ticket about the change: https://trello.com/c/infKb5AQ/3401-enb-suse-liberty-linux-7-adjustments

Fixes # n/a

## Change Type

*Please select the correct option.*

- [ ] **Bug Fix** (a non-breaking change which fixes an issue)
- [ ] **New Feature** (a non-breaking change which adds new functionality)
- [ ] **Documentation Update** (a change which only updates documentation)

## Checklist

*Please check off each item if the requirement is met.*

- [x] I have verified that my code follows RMT's coding standards with `rubocop`.
- [x] I have reviewed my own code and believe that it's ready for an external review.
- [ ] I have provided comments for any hard-to-understand code.
- [ ] I have documented the `MANUAL.md` file with any changes to the user experience.
- [ ] RMT's test coverage remains at 100%.
- [ ] If my changes are non-trivial, I have added a changelog entry to notify users at `package/obs/rmt-server.changes`.

## Other Notes

Please use this space to provide notes or thoughts to the team, such as tips on how to review/demo your changes.

### CODE REVIEW

**Comment by digitaltom** [comment]
@Adnilson @josegomezr fixing the test will unblock all other prs :-) :pray:

File: features/lockfile_spec.rb [hunk 1]
```diff
@@ -1,13 +1,17 @@
-require File.expand_path('../support/command_rspec_helper', __FILE__)
-
 describe 'rmt-cli' do
-  before do
+  around do |example|
+    parent_pidfile = "/tmp/rmt-parent-#{Process.pid}.pid"
+
     `echo long_text_but_not_the_process_id > /tmp/rmt.lock`
```
Comment (josegomezr) [inline features/lockfile_spec.rb:5]:
```suggestion
    File.write('long_text_but_not_the_process_id', '/tmp/rmt.lock')
```

File: features/lockfile_spec.rb [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
     `chown _rmt /tmp/rmt.lock`
-    `/usr/bin/rmt-cli sync > /dev/null &`
+
+    fork {
+      File.write(parent_pidfile, Process.pid)
+      exec "/usr/bin/rmt-cli sync >/dev/null"
+    }
+    example.run
+    Process.kill('TERM', File.read(parent_pidfile).to_i)
```
Comment (josegomezr) [inline features/lockfile_spec.rb:13]:
```suggestion
    Process.kill('TERM', File.read(parent_pidfile).to_i)
    File.delete('/tmp/rmt.lock')
```

**Review by josegomezr — COMMENTED** [review]
IIUC the error correctly, (the lock file stays and breaks the subsequent feature tests) the suggestions should fix it.

File: features/lockfile_spec.rb [hunk 3]
```diff
@@ -1,13 +1,18 @@
...
+    File.delete('/tmp/rmt.lock')
```
Comment (digitaltom) [inline features/lockfile_spec.rb:14]: FileUtils.rm('/tmp/rmt.lock', force: true)

Comment (digitaltom) [inline features/lockfile_spec.rb:14]:
```suggestion
    FileUtils.rm('/tmp/rmt.lock', force: true)
```

File: features/lockfile_spec.rb [hunk 4]
```diff
@@ -1,13 +1,18 @@
-require File.expand_path('../support/command_rspec_helper', __FILE__)
-
 describe 'rmt-cli' do
-  before do
-    `echo long_text_but_not_the_process_id > /tmp/rmt.lock`
+  around do |example|
+    parent_pidfile = "/tmp/rmt-parent-#{Process.pid}.pid"
+
+    File.write('long_text_but_not_the_process_id', '/tmp/rmt.lock')
```
Comment (digitaltom) [inline features/lockfile_spec.rb:5]:
```suggestion
    File.write('/tmp/rmt.lock', 'long_text_but_not_the_process_id')
```

**Pull request merged**